tests/utils/mod.rs

```diff
@@ -2,3 +2,21 @@
 pub mod managed_utils;
 /// A framework to create integration tests of the "raw" mls_group API.
 pub mod mls_utils;
+
+#[allow(unused_macros)]
+macro_rules! ctest_ciphersuites {
```

Comment:
Just a minor point: Can't we just expose this code under some flag as part of `openmls` instead of duplicating it in `/tests` and `/src/test_util`?

Reply:
There's no easy way unfortunately. The integration tests are running against a regular build of the library. So the only "easy" way I see is to require a feature flag to be present for testing, which isn't great.

> I realize that it's necessary, but we might want to think about how to reduce the noise. For example, there seem to be quite a few complaints related to test vector generation.

Yeah, this is getting us a baseline again now that we can look at and see if we want to disable something.